spec/unit/installer/xcode/pods_project_generator/pod_target_installer_spec.rb

```diff
@@ -1075,13 +1075,17 @@ class PodsProjectGenerator
                   snake-umbrella.h
                 )
 
-                copy_files_build_phases = target.copy_files_build_phases.sort_by(&:name)
-                copy_files_build_phases.map(&:name).should == [
-                  'Copy . Public Headers',
+                target.build_phases.map(&:display_name).should == [
```

Comment:
updated test. The test here no longers perform `sort` and checks against _all_ script phase names instead of just the `copy_files_build_phases`.

Verified my change work and verified the test fails without my change in terms of build phase ordering.